packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -19,14 +19,25 @@ import {FocusRing, FocusScope} from '@react-aria/focus';
 // @ts-ignore
 import intlMessages from '../intl/*.json';
 import {ListCollection, useListState} from '@react-stately/list';
-import {Provider, useProviderProps} from '@react-spectrum/provider';
+import {Provider, useProvider, useProviderProps} from '@react-spectrum/provider';
 import React, {ReactElement, useCallback, useEffect, useMemo, useRef, useState} from 'react';
 import styles from '@adobe/spectrum-css-temp/components/tags/vars.css';
 import {Tag} from './Tag';
 import {useFormProps} from '@react-spectrum/form';
 import {useId, useLayoutEffect, useResizeObserver, useValueEffect} from '@react-aria/utils';
 import {useLocale, useLocalizedStringFormatter} from '@react-aria/i18n';
 
+const TAG_STYLES = {
```

Comment:
If the css changes this breaks, right? Shouldn't there be a comment on the CSS and this making that connection?

Reply:
True, but I don't think these change too often, and we do the same type of thing in TableView and ListView. I think if we have a big enough change to styles, we'll notice here.